tool-testsuite/test/org/antlr/v4/test/tool/TestSymbolIssues.java

```diff
@@ -401,4 +401,34 @@ public void testLabelsForTokensWithMixedTypesLRWithoutLabels() {
 
 		testErrors(test, false);
 	}
+
+	@Test public void testUnreachableTokens() {
+		String[] test = {
+				"lexer grammar Test;\n" +
+				"TOKEN1: 'as' 'df' | 'qwer';\n" +
+				"TOKEN2: [0-9];\n" +
+				"TOKEN3: 'asdf';\n" +
+				"TOKEN4: 'q' 'w' 'e' 'r' | A;\n" +
+				"TOKEN5: 'aaaa';\n" +
+				"TOKEN6: 'asdf';\n" +
+				"TOKEN7: 'qwer'+;\n" +
+				"TOKEN8: 'a' 'b' | 'b' | 'a' 'b';\n" +
+				"fragment\n" +
+				"TOKEN9: 'asdf' | 'qwer' | 'qwer';\n" +
```

Comment:
:bulb: Should include tokens with characters that need escapes, to make sure the values are properly escaped in the error messages.

Reply:
Done.